fastlane/Fastfile

```diff
@@ -757,16 +765,15 @@ lane :rubocop do
 end
 
 lane :install_sim do |options|
-  xcode_install_cache_dir = File.expand_path('~/Library/Caches/XcodeInstall')
-  sim_dmg_path = Dir["#{xcode_install_cache_dir}/*.dmg"].first
-  sim_pkg_path = Dir["#{xcode_install_cache_dir}/*.pkg"].first
-  if is_localhost || sim_dmg_path.nil? || sim_pkg_path.nil?
-    sh("bundle exec xcversion simulators --install='iOS #{options[:ios]}'")
+  runtime_list = sh("xcrun simctl runtime list | grep 'iOS #{options[:ios]}' || true")
+  if runtime_list.include?('Ready')
```

Comment:
what's this, feels a bit hacky?

Reply:
I'm just checking if we already have a runtime up and running on the requested machine to avoid an unnecessary runtime download & install.

Reply:
I see, it prints
```
❰tvahter❙~❱✔≻ xcrun simctl runtime list
== Disk Images ==
-- iOS --
iOS 17.5 (21F79) - F4250FD9-3E7E-4AED-B634-8C930CF63BDD (Ready)
iOS 16.4 (20E247) - 55C705E9-2296-423D-8035-8A6723106E23 (Ready)
iOS 15.5 (19F70) - 12AF78C7-AACF-4F12-8E26-20F586C32DE2 (Ready)

Total Disk Images: 3 (24.1G)
```

Reply:
It would be more reliable to use the json option:
```
xcrun simctl runtime list -j
```
which gives
```
❰tvahter❙~❱✘≻ xcrun simctl runtime list -j
{
  "12AF78C7-AACF-4F12-8E26-20F586C32DE2" : {
    "build" : "19F70",
    "deletable" : true,
    "identifier" : "12AF78C7-AACF-4F12-8E26-20F586C32DE2",
    "kind" : "Legacy Download",
    "path" : "\/Library\/Developer\/CoreSimulator\/Profiles\/Runtimes\/iOS 15.5.simruntime",
    "platformIdentifier" : "com.apple.platform.iphonesimulator",
    "runtimeBundlePath" : "\/Library\/Developer\/CoreSimulator\/Profiles\/Runtimes\/iOS 15.5.simruntime",
    "runtimeIdentifier" : "com.apple.CoreSimulator.SimRuntime.iOS-15-5",
    "signatureState" : "Unknown",
    "sizeBytes" : 12385398784,
    "state" : "Ready",
    "version" : "15.5"
  },
…
```

Reply:
Cool, I didn't know it could return a json. Updated 👍